src/models/Draft.ts

```diff
@@ -84,6 +84,7 @@ export class DraftL {
             name: this.data.title,
             //
             appID: this.data.appID,
+            draftID: this.data.id,
```

Comment:
does this change anything ? Is that a minor optimisation to bypass going though the getter ?happy to accept this anyway :)